kolibri/core/assets/src/core-app/constructor.js

```diff
@@ -156,6 +157,7 @@ module.exports = class CoreApp {
     publicMethods.forEach((method) => {
       this[method] = mediator[method].bind(mediator);
     });
+    new HeartBeat(this);
```

Comment:
We instantiate the heartbeat here, giving it a reference to the Kolibri instance.